ReSharper.FSharp/src/FSharp.Psi.Features/src/Formatter/FSharpCodeStylePage.fs

```diff
@@ -42,6 +42,9 @@ type FSharpFormatSettingsKey() =
     
     [<SettingsEntry(true, "Space around delimiter"); DefaultValue>]
     val mutable SpaceAroundDelimiter: bool
+
+    [<SettingsEntry(false, "Preserve end of line"); DefaultValue>]
```

Comment:
I'd probably set it to `true` as default setting so cli tooling works properly with formatted files. Is there a reason to set it to `false` for all users by default?

Reply:
The default fantomas setting is `false`. That is why, but I'm happy to go with `true`.